packages/react-component-library/src/fragments/Masthead/Masthead.tsx

```diff
@@ -0,0 +1,150 @@
+  Logo?: React.ComponentType
+  navItems?: NavItemTypes[]
+  NotificationsPopoverContent?: JSX.Element
+  onSearch?: (term: string) => void
+  searchPlaceholder?: string
+  title: string
+  unreadNotification?: boolean
+  user?: UserType
+}
+
+export const Masthead: React.FC<MastheadProps> = ({
+  homeLink,
+  LinkComponent = Link,
+  Logo = DefaultLogo,
+  navItems,
+  NotificationsPopoverContent,
+  onSearch,
+  searchPlaceholder = '',
+  title,
+  unreadNotification,
+  user,
+}) => {
+  const [showSearch, setShowSearch] = useState(false)
+  const [showNotifications, setShowNotifications] = useState(false)
+  const searchButtonRef = useRef<HTMLButtonElement>(null)
+  const mastheadContainerRef = useRef<HTMLDivElement>(null)
+  const [containerWidth, setContainerWidth] = useState(0)
+
+  const toggleSearch = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => {
+    event.currentTarget.blur()
+    const newShowSearch = !showSearch
+
+    // if opening the searchbar then get the container width and set that
+    // as the width of the searchbar so that it does not spill
+    // over to other parts of the page, such as the sidebar.
+    if (newShowSearch === true) {
+      setContainerWidth(mastheadContainerRef.current.offsetWidth)
+    }
+
+    setShowSearch(!showSearch)
+  }
+
+  return (
+    <div
+      className={classNames('rn-masthead', {
+        'rn-masthead--show-search': showSearch,
+        'rn-masthead--show-notifications': showNotifications,
+      })}
+      data-testid="masthead"
+      ref={mastheadContainerRef}
+    >
+      <div className="rn-masthead__main">
+        <LinkComponent {...homeLink} className="rn-masthead__service-name">
+          <Logo />
+          <span
+            className="rn-masthead__title"
+            data-testid="masthead-servicename"
+          >
+            {title}
+          </span>
+        </LinkComponent>
+
+        <div className="rn-masthead__options">
+          {onSearch && (
+            <>
+              <button
+                className={`rn-masthead__option rn-searchbar__btn ${
+                  showSearch ? 'is-active' : 'is-inactive'
+                }`}
+                onClick={toggleSearch}
+                ref={searchButtonRef}
+                data-testid="masthead-search-button"
+              >
+                <SearchIcon />
+              </button>
+              <svg className="rn-masthead__vertical-separator">
+                <line x1="0" y1="14" x2="0" y2="44" />
+              </svg>
+            </>
+          )}
+
+          {NotificationsPopoverContent && (
+            <NotificationPanel
+              buttonClassName="rn-masthead__option"
+              className="rn-masthead__notification"
+              data-testid="masthead-notifications"
+              notificationPlacement="below"
+              onHide={() => setShowNotifications(false)}
+              onShow={() => setShowNotifications(true)}
+              scheme="light"
```

Comment:
Should this prop value be hard coded?

Reply:
Its telling it to use the light version of notifications, compared to sidebar using the dark so I think it's ok